KarinaKlevzhits/hometask_5/#4.8.sql

```diff
@@ -0,0 +1,6 @@
+DELETE FROM film_category WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar');
+DELETE FROM film_actor WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar');
+DELETE FROM payment WHERE rental_id in (SELECT rental_id FROM rental WHERE inventory_id in (SELECT inventory_id FROM inventory WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar')));
+DELETE FROM rental WHERE inventory_id in (SELECT inventory_id FROM inventory WHERE film_id in (SELECT film_id FROM film WHERE title = 'Aladdin Calendar'));
+DELETE FROM inventory WHERE film_id in (SELECT film_id FROM film  WHERE title = 'Aladdin Calendar');
+DELETE FROM film WHERE title='Aladdin Calendar';
```

Comment:
Can you try to change tables with foreign key to delete all related record using only one request
alter table film_actor
  drop constraint film_actor_foreign_for_film_id_example,
  add constraint fim_id_fkey
    foreign key (film_id)
    references film (film_id)
    on delete cascade
    not valid;